app/main.py

```diff
@@ -1,3 +1,7 @@
 def count_occurrences(phrase: str, letter: str) -> int:
-    # write your code here
-    pass
+    counter = 0
+    for char in phrase:
+        if char.lower() == letter.lower():
+            counter += 1
+
```

Comment:
There is an indentation issue here. The return statement should be aligned with the for loop to ensure it executes after the loop completes. Currently, it is not properly indented, which will cause a syntax error.